src/main/java/jdbc/mapping/AssociationFieldMapping.java

```diff
@@ -21,45 +21,45 @@ public <T> boolean supports(Class<T> entityType) {
 
     @Override
     public <T> T getRow(ResultSet resultSet, Class<T> entityType) throws IllegalAccessException, SQLException {
-        final List<Field> fields = getPersistentFields(entityType);
-        final Class<?> joinColumnType = getJoinColumnType(entityType);
-
+        final EntityTable entityTable = new EntityTable(entityType);
+        final Class<?> associationColumnType = entityTable.getAssociationColumnType();
         final T entity = new InstanceFactory<>(entityType).createInstance();
-        final List<Object> list = getList(fields, entity);
+        final List<Object> collection = getCollection(entity, entityTable);
 
         do {
-            list.add(getChildEntity(resultSet, joinColumnType, fields, entity));
+            collection.add(getChildEntity(resultSet, associationColumnType, entity, entityTable));
         } while (resultSet.next());
```

Comment:
사소하지만, 코드를 읽는 사람 입장에서는 한 호흡이 분리되면 좀 더 읽기 편할 것 같아요 :)
```suggestion
        do {
            final Object childEntity = createChildEntity(resultSet, associationColumnType, entity, entityTable);
            collection.add(childEntity);
        } while (resultSet.next());
```